src/components/landingComponents/LandingHero.vue

```diff
@@ -44,5 +52,42 @@
                 </svg>
             </a>
         </div>
+
+        <div class="background-circle-bb bg-dt-v-light z-0"></div>
+        <div class="background-diamond-bb bg-dt-v-light z-0"></div>
```

Comment:
I suspect that these shapes appear in front of EVERYTHING. In order to make it work, you'll need to change everything to absolute so be careful.